translate/convert/po2rc.py

```diff
@@ -16,47 +16,280 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, see <http://www.gnu.org/licenses/>.
-
 """Convert Gettext PO localization files back to Windows Resource (.rc) files.
 
 See: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/commands/rc2po.html
 for examples and usage instructions.
 """
 
+import codecs
+from collections.abc import Iterable
+from translate.storage.rc import generate_popup_caption_name,\
+    generate_popup_pre_name, generate_menu_pre_name
+
 
 from translate.convert import convert
 from translate.storage import po, rc
 
+NL = "\n"
```

Comment:
Should this be `\r\n` if it's a windows specific format?

Reply:
Probably yes. It's coming from https://github.com/translate/translate/commit/86e90ea3760e5b00f19c37e022d399de87262342 and I really didn't touch the logic there...